store/storetest/link_metadata_store.go

```diff
@@ -228,10 +228,10 @@ func testLinkMetadataStoreTypes(t *testing.T, ss store.Store) {
 		require.IsType(t, &opengraph.OpenGraph{}, received.Data)
 		assert.Equal(t, *(metadata.Data.(*opengraph.OpenGraph)), *(received.Data.(*opengraph.OpenGraph)))
 
-		result = <-ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
-		require.Nil(t, result.Err)
+		linkMetadata, err := ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
```

Comment:
probably you can use here the `received, err := ...` and remove the received assignation some lines later.